packages/core/contracts/data-verification-mechanism/implementation/Staker.sol

```diff
@@ -276,8 +276,8 @@ abstract contract Staker is StakerInterface, Ownable, Lockable, MultiCaller {
     }
 
     // Updates an account internal trackers.
-    function _updateTrackers(address voterAddress) internal virtual {
```

Comment:
I re-named this variable throughout the contracts for consistency and simplification reasons.